<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>575</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1006</width>
    <height>575</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1006</width>
    <height>575</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40, 38, 38);</string>
     </property>
     <widget class="QPushButton" name="logo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>491</width>
        <height>401</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>491</width>
        <height>401</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>421</width>
        <height>401</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 1px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/Logo.png</normaloff>image/Logo.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>550</width>
        <height>520</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(199, 194, 194);</string>
     </property>
     <widget class="QLabel" name="welcome">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(177, 3, 3);</string>
      </property>
      <property name="text">
       <string>Welcome</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelEmail">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(177, 3, 3);</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputEmail">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>280</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid  #B10303;
border-radius: 5px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPass">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>350</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(177, 3, 3);</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputPass">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>380</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid  #B10303;
border-radius: 5px;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLogin">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>490</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(177, 3, 3);
color: white;
font-size: 20px;
border-radius: 5px;</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="error">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>180</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
